newsroom/agenda/service.py

```diff
@@ -0,0 +1,118 @@
+from datetime import datetime
+
+from newsroom.auth.utils import get_user_from_request, get_company_from_request
+from newsroom.agenda.agenda import (
+    is_events_only_access,
+    build_agenda_query,
+    nested_query,
+    planning_items_query_string,
+    aggregations,
+    remove_fields,
+    PLANNING_ITEMS_FIELDS,
+)
+from newsroom.agenda.model import FeaturedResourceModel
+from newsroom.utils import get_local_date
+from newsroom.template_filters import is_admin
+
+from superdesk.flask import abort
+from superdesk.core.resources import AsyncResourceService
+from superdesk.utc import local_to_utc
+from superdesk.utils import ListCursor
+
+
+class FeaturedService(AsyncResourceService[FeaturedResourceModel]):
+    resource_name = "agenda_featured"
+
+    async def on_create(self, docs):
+        """
+        Add UTC from/to datetimes on save.
+        Problem is 31.8. in Sydney is from 30.8. 14:00 UTC to 31.8. 13:59 UTC.
+        And because we query later using UTC, we store those UTC datetimes as
+        display_from and display_to.
+        """
+        for item in docs:
+            date = datetime.strptime(item._id, "%Y%m%d")
```

Comment:
When removing the `_id` field you defined on the resource, you would then use the standard `item.id` field instead